src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverseRob = {};
+
+  for (const key in robot) {
+    if (inverseRob.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    inverseRob[robot[key]] = key;
```

Comment:
It would be better to add a blank before for a better visual

Reply:
agree